lib/bolt/logger.rb

```diff
@@ -91,6 +91,14 @@ def self.configured?
       Logging.logger[:root].appenders.any?
     end
 
+    def self.stream
+      @stream
+    end
+
+    def self.stream=(stream)
+      @stream = stream
+    end
```

Comment:
Yeah - I think I prefer this because having instance variables on modules already feels weird, and explicitly defining a getter and setter...makes it feel better? It to me reads as more clear that we know we're doing something weird but we're doing it anyway.